### PR TITLE
Fix edgecore cpu usage issue

### LINKED ISSUE #1205: edgecore cpu usage rate will increase by container's num
**What happened**:
edgecore cpu usage rate will increase by container's num。
cpu usage rate over 10% when 6 pods run 
**What you expected to happen**:
edgecore's cpu usage rate below than kubelet 
**How to reproduce it (as minimally and precisely as possible)**:
create some pod and check node's cpu usage rate
**Anything else we need to know?**:

**Environment**:
- KubeEdge version(e.g. `cloudcore/edgecore --version`):  

**CloudSide Environment**:
- Hardware configuration (e.g. `lscpu`):
- OS (e.g. `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Go version (e.g. `go version`): 
- Others:

**EdgeSide Environment**:
- edgecore version (e.g. `edgecore --version`):
- Hardware configuration (e.g. `lscpu`):
- OS (e.g. `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Go version (e.g. `go version`): 
- Others:

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:
**Before the fix:** 

- when 40 pods running in edge node, the cpu usage of edgecore will reach 40%.

**With this fix:** 

- when 100 pods running in edge node, the cpu usage of edgecore is between 3%~5%.

**Which issue(s) this PR fixes**:

Fixes #1205

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Fix edgecore cpu usage issue
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ git:
 
 sudo: required
 
-dist: trusty
```
Comment (fisherxu) [inline .travis.yml]: image
When using ubuntu 14.04(trusty), the matching docker version is 17.9. This error is reported when starting the pod/container, but the status of the Pod is Running.
So the previous e2e runs have problems, now upgrade to 16.04, the pod is running normally.

File: .travis.yml [hunk 2]
```diff
@@ -78,6 +79,7 @@ jobs:
     - stage: "Test on arm64"
       name: "build, smallbuild"
       arch: arm64
+      dist: trusty
```
Comment (fisherxu) [inline .travis.yml]: ditto

File: .travis.yml [hunk 3]
```diff
@@ -37,6 +37,7 @@ jobs:
     - stage: "Test on amd64"
       name: "build, smallbuild, crossbuild"
       arch: amd64
+      dist: trusty
```
Comment (fisherxu) [inline .travis.yml]: There's some crossbuild error with 16.04, so using 14.04 for workaround here.
/cc @kevin-wangzefeng @chendave @kadisi

Comment (kevin-wangzefeng) [reply .travis.yml]: I'm fine to get the fix in with this restriction, please create a follow-up issue to track this.

Comment (fisherxu) [reply .travis.yml]: https://github.com/kubeedge/kubeedge/pull/1183  There is already a WIP PR to solve this issue, but it‘s a little complicated that relate to the vm image of travis.

Comment (kevin-wangzefeng) [reply .travis.yml]: OK

File: edge/pkg/edged/pleg/generic.go [hunk 4]
```diff
@@ -21,484 +21,29 @@ This file is derived from K8S Kubelet code with reduced set of methods
...
-	"k8s.io/kubernetes/pkg/kubelet/prober"
-	"k8s.io/kubernetes/pkg/kubelet/status"
-	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
-
-	"github.com/kubeedge/kubeedge/edge/pkg/edged/podmanager"
 )
 
 //GenericLifecycle is object for pleg lifecycle
 type GenericLifecycle struct {
 	pleg.GenericPLEG
```
Comment (kadisi) [inline edge/pkg/edged/pleg/generic.go]: delete `GenericLifecycle ` struct , and use `pleg.GenericPLEG` directly?

Comment (fisherxu) [reply edge/pkg/edged/pleg/generic.go]: Good idea, have done.

**Pull request merged**